course_contents/2_intro_to_python/notes/02. Loops.md

```diff
@@ -6,15 +6,18 @@
 
 **Extra resources**
 [What is a loop?](https://python.tecladocode.com/3_continuous_countries/1_loops.html#what-is-a-loop)
+[Why do we need loops?](https://www.teclado.com/30-days-of-python/python-30-day-6-for-loops#why-do-we-need-loops)
 
 ### The `while` loop
-- while loop runs an indefinite number of times i.e. while loop runs until the given conditional becomes False
-- syntax for writing the while loop:
-    1. use the `while` keyword at the beginning of the loop
-    2. the keyword should be followed with a correctly written conditional with a colon at the end of it
-    3. indented block of code that will run until the loop breaks
+- `while` loop runs an indefinite number of times i.e. `while` loop runs until the given conditional becomes `False`
+- syntax for the `while` loop:
```

Comment:
Don't think you need this line!

Reply:
Deleted the line and modified the first one to make usage of the name of the loop being already mentioned